frontend/src/components/note/NoteCard.vue

```diff
@@ -69,61 +69,41 @@ onBeforeMount(() => {
       <Divider type="solid" />
     </template>
     <template #content>
-      <div class="grid grid-cols-12 gap-4 nested-grid">
-        <!-- Left column -->
-        <div class="col-span-12 md:col-span-6 lg:col-span-3">
-          <div class="grid grid-cols-12 gap-4">
-            <p class="col-span-12">
-              <span class="key font-bold">Date:</span>
-              {{ note.createdAt ? formatDateShort(note.createdAt) : undefined }}
-            </p>
-          </div>
+      <div class="grid grid-cols-4 gap-4">
+        <p>
+          <span class="key font-bold">Date:</span>
+          {{ noteHistory.createdAt ? formatDateShort(noteHistory.createdAt) : undefined }}
```

Comment:
wouldn't we always have a created at because of `generateCreateStamps()` in the api?